src/ink.tsx

```diff
@@ -0,0 +1,182 @@
+import React, {ReactNode} from 'react';
+import throttle from 'lodash.throttle';
+import logUpdate, {LogUpdate} from 'log-update';
+import isCI from 'is-ci';
+import reconciler from './reconciler';
+import createRenderer, {InkRenderer} from './renderer';
+import signalExit from 'signal-exit';
+import {createNode, DOMNode} from './dom';
+import instances from './instances';
+import App from './components/App';
+
+export interface InkOptions {
+	stdout: NodeJS.WriteStream;
+	stdin: NodeJS.ReadStream;
+	debug: boolean;
+	exitOnCtrlC: boolean;
+	waitUntilExit?: () => Promise<any>;
```

Comment:
Don't use `any`. Also applies to all the other places `any` is used. Try to use a more explicit type.

Reply:
I removed any everywhere except for in Color component.